index.js

```diff
@@ -184,9 +184,9 @@ module.exports = function (content) {
         urls[format] = path.join(pub, url).replace(/\\/g, '/');
 
         if (generatorConfiguration.dest) {
-          self.emitFile(urls[format], res[format]);
+          this.emitFile(urls[format], res[format]);
```

Comment:
Indeed you are right, the arrow function allows this to work. I just looked only at the latest commit and it didn't show the function update.

All is O.K. then, the tests pass.